### ISSUE TITLE
Bug - LDAP users and dashboards

### ISSUE THREAD

**Issue by pblasquez** [librenms/librenms#6489]
DO NOT DELETE THIS INFORMATION.

> Please read this information carefully.

GitHub issues is for  bugs, please do not post issues asking for help or how to do X, Y or Z. 
You can use our irc channel ##librenms on freenode to ask questions or our [community site](https://community.librenms.org).
If you have a feature request please post this on our [community site](https://community.librenms.org/c/feature-requests).

Please confirm each of the sections below by putting an `x` in the box like `[x]`.

- [x] Is your install up to date? [Updating your install](http://docs.librenms.org/General/Updating/)
      Please do not submit an issue if your install is not up to date within the last 24 hours or on a stable monthly release.
- [x] Please include all of the information between the `====================================` section of `./validate.php` which you can run from the cli.
==========================================================
Component | Version
--------- | -------
LibreNMS  | d77186ddc68bf4eedef8e20fab2e7883138617bc
DB Schema | 186
PHP       | 5.4.16
MySQL     | 5.5.44-MariaDB-log
RRDTool   | 1.5.999
SNMP      | NET-SNMP 5.7.2
==========================================================

Hello,

TLDR; The overview/dashboards page appears to currently be broken for LDAP users, possibly only those with large AD environments.  The issue seems to be with the get_dashboards() function in html/includes/functions.inc.php.

Details:

I am using LDAP authentication. The 'users' table used to include an entry for all users that have successfully authenticated.  After my last update, the 'users' table has been truncated of all LDAP users, leaving only the original 'admin' user with a user_id of 1.

What led me to discover the above was that the 'overview' page was timing out.  I isolated the timeout to the get_dashboards() function, and found that it was referencing the user_ids in the dashboard table, which led me to the discovery that there are no longer LDAP users in the table.

Wanting to put my overview page back into service, I deleted all Default dashboards and assigned a user_id of 1 to the rest.  Unfortunately, this was only a temporary fix.  I created a test dashboard and it assigned my original user_id but did not create an entry in the 'users' table. It also recreated my user's Default dashboard.  As soon as I did this, the timeout returned.

My current fix is to comment out the get_user() function within get_dashboards() in html/includes/functions.inc.php, replacing it with the admin user details.

**Comment by pblasquez** [comment]
Ok, I think I found the issue, it is actually due to the ownership of the dashboards by user_id 1, the admin account.

I changed the user_id for the shared dashboards to my own LDAP ID and everything works fine now.  Hopefully this helps someone with the same situation.
